src/loaddb/load_server_loader.cpp

```diff
@@ -124,16 +124,17 @@ namespace cubload
 	return found;
       }
 
+#if defined(SERVER_MODE)
```

Comment:
server_class_installer::locate_class is only called in server mode.
In sa mode, functions in the sa_class_installer namespace are called.

In css_conn_entry, client_type is defined only in SERVER_MODE,
so to prevent warning message output, I also specified SERVER_MODE in server_class_installer::locate_class.

```
/* src/connection/connection_defs.h */

/*
 * This data structure is the interface between the client and the
 * communication software to identify the data connection.
 */
typedef struct css_conn_entry CSS_CONN_ENTRY;
struct css_conn_entry
{
  ...
#if defined(SERVER_MODE)
  ...
  BOOT_CLIENT_TYPE client_type;
  ...
#else
  ...
#endif
...
```